spec/licensee/matchers/nu_get_matcher_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
```

Comment:
The name of this file is funny due to rubocop wanting "NuGet" -> "nu_get" - if there's a way to suppress that, I'm happy to change it. Also fine as is, your call.

Reply:
Does it complain about `nuget`? That would seem preferable to `nu_get`, and matches the library file name.

Reply:
Yes, that's the issue:
```
spec/licensee/matchers/nuget_matcher_spec.rb:3:1: C: RSpec/FilePath: Spec path should end with licensee/matchers/nu_get*_spec.rb.
```
 I don't know my way around rubocop to suppress that.

Reply:
I see, it looks like rubocop wants spec naming to be derived from the tested class name in a specific way and it's possible to override, see https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FilePath but maybe not worth overriding.

Reply:
Going to leave it unless someone feels strongly.